scripts/filter_inputs/smarts_queries.csv

```diff
@@ -0,0 +1,13 @@
+#A1Ac2ccc(Cl)cc2C2(CCN(c3cncc4ccccc34)C2=O)A1,tetralinlike-5spiro-isoquinoline
```

Comment:
can we delete this file now that we have the copy in `data/`?

Reply:
yes, forgot to `git rm`. see https://github.com/choderalab/covid-moonshot-ml/pull/42/commits/00e08fbd3e8560b59f94f70910c7496a23544282